### DISCUSSION TITLE
Showing an annulus mesh

### DISCUSSION THREAD

**Discussion by amnchacon** [firedrakeproject/firedrake#3339] [Firedrake support]
Hello good morning. I'm starting to learn the Firedrake code because I'll use it for my thesis work and I don't know how to show an annulus mesh, or a circle with a radial extruded region. In the Firedrake User Manuel, specifically 37 page it is shown an annulus mesh but it's not shown the code to print it. Could you help me? ... I already tried with some alternatives: plot, triplot, obtaining the spatial coordinates and  I don't get yet.

**Comment by amnchacon** [comment]
I want to resolve an specific problem, but I like to prove by parts, and firstly I want to see at least the mesh....
![Captura de tela de 2024-01-20 09-06-22](https://github.com/firedrakeproject/firedrake/assets/146268309/c772465b-9412-4587-ac37-ce872bdac3de)

**Comment by colinjcotter** [comment]
The primary visualisation tool for Firedrake is pvd files in Paraview.

https://www.firedrakeproject.org/visualisation.html

**Reply by amnchacon** [reply]
Oh, Nice! I'll read it. If any questions arise, I'll write here again. Thanks for the information.
